src/main/webapp/js/build-message-div.js

```diff
@@ -42,7 +42,71 @@ function formatTimestamp(messageTime){
           timestamp.getHours()+":"+timestamp.getMinutes()+":"+timestamp.getSeconds();
 }
 
-function buildMessageInTimeline(message, flip){
+function createDeleteButton(deleteConfirmationDiv){
+  var btn = document.createElement('button');
+  btn.type='button';
+  btn.classList.add('delete');
+  var icon = document.createElement('i');
+  icon.classList.add('fa','fa-trash');
```

Comment:
Did not know you could just add all the classes at once, that's very useful!

```suggestion
  icon.classList.add('fa','fa-trash','m-2');
```
I tried with this margin, it looks a little bit better because the button has some margin!